components/logo/Logo.module.css

```
@@ -0,0 +1,29 @@
+.main {
+  font-weight: 900;
+  font-size: 8.5em;
+  line-height: 1em;
+  background-image: linear-gradient(
+      var(--secondary-color),
+      var(--secondary-color)
+    ),
+    linear-gradient(var(--secondary-color), var(--secondary-color));
+  background-repeat: no-repeat;
+  background-size: 3px 55%, 64% 3px;
+  background-position: right bottom, right bottom;
+  padding: 0 0.1em;
+}
+
+.small {
+  font-weight: 600;
+  font-size: 2.7em;
+  line-height: 1em;
+  background-image: linear-gradient(
+      var(--secondary-color),
+      var(--secondary-color)
+    ),
+    linear-gradient(var(--secondary-color), var(--secondary-color));
+  background-repeat: no-repeat;
+  background-size: 2px 55%, 64% 2px;
+  background-position: right bottom, right bottom;
+  padding: 0 0.12em;
+}
```

There is some CSS which is identical.
You can create a basic class:

```suggestion
.logo {
  line-height: 1em;
  background-image: linear-gradient(
      var(--secondary-color),
      var(--secondary-color)
    ),
    linear-gradient(var(--secondary-color), var(--secondary-color));
  background-repeat: no-repeat;
  background-position: right bottom, right bottom;
}

.main {
  font-weight: 900;
  font-size: 8.5em;
  background-size: 3px 55%, 64% 3px;
  padding: 0 0.1em;
}

.small {
  font-weight: 600;
  font-size: 2.7em;
  background-size: 2px 55%, 64% 2px;
  padding: 0 0.12em;
}
```

Or say that one size is default and `small` overrides some properties.